packages/runtime/container-runtime/src/test/gc/garbageCollection.spec.ts

```diff
@@ -770,22 +709,6 @@ describe("Garbage Collection Tests", () => {
 				"GarbageCollector:SweepReadyObject_Revived",
 				"GarbageCollector:SweepReadyObject_Changed",
 				"GarbageCollector:SweepReadyObject_Loaded",
-				true, // expectDeleteLogs
-			);
-		});
-
-		describe("SweepReady events - Delete log disabled (summarizer container)", () => {
```

Comment:
I'm both proud and annoyed that we had this test. Do we have too many ways to configure GC? 🤔

Reply:
We probably do. There are a lot of feature flags. We should do a clean up soon!